source/server/admin/prometheus_stats.cc

```diff
@@ -236,6 +247,10 @@ uint64_t PrometheusStatsFormatter::statsAsPrometheus(
       outputStatType<Stats::Gauge>(response, used_only, regex, gauges,
                                    generateNumericOutput<Stats::Gauge>, "gauge", custom_namespaces);
 
+  metric_name_count +=
+      outputStatType<Stats::TextReadout>(response, used_only, regex, text_readouts,
+                                         generateTextReadoutOutput, "gauge", custom_namespaces);
```

Comment:
document why arbitrary text is OK for a 'gauge' in Prometheus?

Reply:
Done